packages/web/src/components/result/ResultCard.js

```diff
@@ -32,27 +32,22 @@ class ResultCard extends Component {
 	constructor(props) {
 		super(props);
 
+		let currentPage = 0;
+		if (this.props.defaultPage) {
```

Comment:
we can check here if the defaultPage >= 0 to be valid. (after setting it as -1 as the invalid value)

Reply:
@metagrover Sounds good. I'll make the change.